ArduCopter/mode.cpp

```diff
@@ -323,12 +327,12 @@ bool Copter::Mode::takeoff_triggered(const float target_climb_rate) const
         // can't takeoff unless we want to go up...
         return false;
     }
-#if FRAME_CONFIG == HELI_FRAME
-    if (!motors->rotor_runup_complete()) {
-        // hold heli on the ground until rotor speed runup has finished
+//  This now applies to multi's and Heli's
+    if (motors->get_spool_mode() == AP_Motors::THROTTLE_UNLIMITED) {
```

Comment:
Shouldn't this be `!=` instead of `==`?

Reply:
Yes, you are correct.  Good catch.  I will fix that when I get home.  
V/R,
Bill